docusaurus/docs/proxying-api-requests-in-development.md

```diff
@@ -31,12 +32,11 @@ Fetch API cannot load http://localhost:4000/api/todos. No 'Access-Control-Allow-
 
 Keep in mind that `proxy` only has effect in development (with `npm start`), and it is up to you to ensure that URLs like `/api/todos` point to the right thing in production. You don’t have to use the `/api` prefix. Any unrecognized request without a `text/html` accept header will be redirected to the specified `proxy`.
 
-The `proxy` option supports HTTP, HTTPS and WebSocket connections.<br>
-If the `proxy` option is **not** flexible enough for you, alternatively you can:
+The `proxy` option supports HTTP, HTTPS and WebSocket connections.
+
+The `proxy` can also be [configured manually](#configuring-the-proxy-manually). This requires more setup but allows for greater control than the `package.json` `proxy` simple option.
 
-- [Configure the proxy yourself](#configuring-the-proxy-manually)
-- Enable CORS on your server ([here’s how to do it for Express](https://enable-cors.org/server_expressjs.html)).
-- Use [environment variables](adding-custom-environment-variables.md) to inject the right server host and port into your app.
+The alternative to proxying is talking directly to a remote API. For static sites, or those served from different hosts/ports than the API they are talking to, then a proxy is not only not needed but not desirable. For those cases, the remote server usually has CORS enabled ([here’s how to do it for Express](http://enable-cors.org/server_expressjs.html)) to respond to requests. The app will need a way to specify what the API endpoint is though. Use [environment variables](adding-custom-environment-variables.md) to inject the right server host and port into your app to specify those. This has the added benefit of being able to have different values and inject them at run or build time.
```

Comment:
This clarifies why and what using using CORS with env vars is for, since that is the main part that is not clear.